std_msgs/Header header

uint8 LEFT  = 0
uint8 RIGHT = 1
uint8 foot_index # LEFT/RIGHT foot

geometry_msgs/Pose pose  # Foot pose
